File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_1.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    if (a == 0) && (b != 0){
        fmt.Print ("Uravnenie ne imeet resheniyi")
    }  else if (a == 0) && (b == 0) {
        fmt.Print ("Uravnenie imeet beskonechnoe mnojestvo resheniyi")
    } else {
        fmt.Print (b / a)
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_1.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
if (a == 0) & (b != 0)
    puts ('yravnenie ne imeet resheniyi')
elsif (a == 0) & (b == 0)
    puts ('yravnenie imeet besconechnoe kol-vo resheniyi')
else
    x = b / a
    puts ("#{x}")
end